release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+# Overview of jBallerina 1.2.0
+
+JBallerina 1.2.0 is the first major Q1 release of 2020 and it includes a new set of features and significant improvements to the compiler, runtime, standard library and developer tooling. It is based on Stable Language Spec version 2020R1.
+
+# Highlights
+
+- Redesigned XML, Stream and Query support
+- Based on a stable language specification: 2020R1
+- Improved Compilation time.
+- Includes a number of important bug fixes
+- Added cookies and trailer support in HTTP.
+- Revamped Cache module.
+- Redesigned oneof field and map field support in gRPC and Added client retry support for gRPC blocking unary calls.
+- Added custom serializer/deserializer and Avro support in the Kafka connector. 
+- Deprecated previous JDBC connector and introduced new Database connectors which are now in a preview state.
+- Introduced new client connectors (email, slack and azure time-series).
+- Changed organization names of all supported connectors.
+- Support to mock functions when writing tests
+- Test and code coverage report
+- Java binding generator tool
```

Comment:
This is a bit misleading right? The tool generates Ballerina bindings for Java APIs. "Java binding generator" gives off a completely opposite meaning

Reply:
@IrushiL

Reply:
Shall we go with "New tool to generates Ballerina bindings for Java APIs."

Reply:
Fixed